src/components/header/Logo.js

```diff
@@ -0,0 +1,8 @@
+import { _querySelector } from "../../utils/my-query-selector.js";
+
+const $haederLogo = _querySelector(".container-header_logo");
+const setHeaderLogo = () => {
+  $haederLogo.addEventListener("click", () => window.location.reload());
```

Comment:
```suggestion
  $haederLogo.addEventListener("click", () => location.reload());
```
window를 생략할 수 도 있어요~